packages/geo/src/lib/layer/shared/layers/vector-layer.ts

```diff
@@ -452,12 +460,29 @@ export class VectorLayer extends Layer {
         ? new olProjection({ code: paramsWFS.srsName })
         : proj;
       const currentExtent = olproj.transformExtent(extent, proj, wfsProj);
+      const ogcFilters = (options as OgcFilterableDataSourceOptions).ogcFilters;
```

Comment:
It would be a nice addition to type at the parameter level of the customWFSLoader method and the return of the method which should be a FeatureLoader. This would allow us to add robustness and make it easier to understand for those unfamiliar with it, like me.